src/lib/containers/SynapseNavDrawer.tsx

```diff
@@ -176,7 +176,7 @@ export const SynapseNavDrawer: React.FunctionComponent<SynapseNavDrawerProps> =
 
   return (
     <div className="SynapseNavDrawer">
-      <Drawer variant="permanent" className='SynapseNavDrawerMenu'>
+      <Drawer variant="permanent" className={`SynapseNavDrawerMenu ${isOpen ? 'tempDrawerOpen' : ''}`}>
```

Comment:
add a css class to the permanent drawer when the temp drawer is open so I can override the z-index in this case (but not in general, since there are other modals in the app!)